ci-operator/config/openshift/cluster-config-operator/openshift-cluster-config-operator-release-4.6.yaml

```diff
@@ -19,7 +19,7 @@ images:
   inputs:
     bin:
       as:
-      - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
+      - registry.svc.ci.openshift.org/ocp/builder:golang-1.13
     ocp_builder_golang-1.13:
```

Comment:
I think you need to drop this automated edit `ocp_builder_golang-1.13` https://deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/9833/rehearse-9833-pull-ci-openshift-cluster-config-operator-master-images/1283044964091039744#1:build-log.txt%3A24

Reply:
Good catch, thx! Should be fixed now.